packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -15,6 +15,7 @@ import {
   getErrorMessageForTypedFieldAndValue,
 } from './typedListHelpers';
 import { ListItemTopBar, ObjectWidgetTopBar, colors, lengths } from 'netlify-cms-ui-default';
+import { compileStringTemplate } from 'netlify-cms-core/dist/esm/lib/stringTemplate';
```

Comment:
Slightly off topic, but if this PR is to be merged, I wonder if we could move `compileStringTemplate` to the util package, or pass it to widgets as a props. I can see this being useful in the `relation` widget as well!

Reply:
I think it would make sense to move it to the util package

Reply:
See https://github.com/netlify/netlify-cms/pull/3659.
Once that is merged we can use `compileStringTemplate` from the util library - I don't think we want widgets to be dependant on the core.